src/NzbDrone.Core/Indexers/Newznab/NewznabSettings.cs

```diff
@@ -16,7 +16,7 @@ public class NewznabSettingsValidator : AbstractValidator<NewznabSettings>
         private static readonly string[] ApiKeyWhiteList =
```

Comment:
Not related to your changed, but since we renamed `Blacklist` -> `Blocklist` I think this should be renamed to `AllowList`.